actor/image_upload/src/lib.rs

```diff
@@ -24,11 +27,68 @@ const MAX_IMAGE_SIZE: usize = 200 * 1024 * 1024; // 200MB
 const IMAGE_BUCKET: &str = "alias_images";
 
 #[allow(dead_code)]
-#[allow(clippy::new_without_default)]
-pub mod wasmcloud_interface_blobstore {
+mod wasmcloud_interface_blobstore {
     include!(concat!(env!("OUT_DIR"), "/gen/blobstore.rs"));
 }
 
+#[allow(dead_code)]
+mod wasmcloud_interface_httpserver {
+    include!(concat!(env!("OUT_DIR"), "/gen/httpserver.rs"));
+}
```

Comment:
Is this the new code to use locally generated interfaces?

Reply:
yes